src/ert/logging/__init__.py

```diff
@@ -9,10 +9,17 @@
 
 class TimestampedFileHandler(FileHandler):
     def __init__(self, filename: str, *args, **kwargs) -> None:
+
         filename, extension = os.path.splitext(filename)
         filename = (
             f"{filename}-"
             f"{datetime.now().strftime(datetime.now().strftime('%Y-%m-%dT%H%M'))}"
             f"{extension}"
         )
+
+        if "use_log_dir_from_env" in kwargs:
```

Comment:
I suggest removing this line and putting a default value of False in the pop method below.